ui/src/pages/OBProxy/New/index.tsx

```diff
@@ -63,9 +63,9 @@ export default function New() {
       style={{ paddingBottom: 50 }}
     >
       <Form onFinish={submit} form={form}>
-        <Space direction="vertical">
+        <Space style={{width:'100%'}} size={'large'} direction="vertical">
           <BasicConfig form={form} />
-          <DetailConfig form={form} />
+          <DetailConfig />
```

Comment:
How can values in `DetailConfig` be transmitted to `form`?

Reply:
Just add the `name` to the `Form.Item` in `DetailConfig`. I originally passed the `form` thinking that it would be needed in this component, but it didn't.